src/plugins/navcurrent/navcurrent.js

```diff
@@ -74,14 +74,15 @@ var componentName = "wb-navcurr",
 						// Pre-process the breadcrumb links
 						localBreadcrumbLinksArray = [];
 						localBreadcrumbLinksUrlArray = [];
-						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "a" );
+						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "li" );
 						len = localBreadcrumbLinks.length;
-						for ( i = 0; i !== len; i += 1 ) {
-							link = localBreadcrumbLinks[ i ];
-							linkHref = link.getAttribute( "href" );
-							if ( linkHref.length !== 0 && linkHref.charAt( 0 ) !== "#" ) {
-								localBreadcrumbLinksArray.push( link );
-								localBreadcrumbLinksUrlArray.push( link.hostname + link.pathname.replace( /^([^\/])/, "/$1" ) );
+						if ( len && len > 0 ) {
```

Comment:
You can just test is len is positive, no need to check if his value is greater than 0
```
if ( len ) {
```